src/ciphers/aes_cbc.rs

```diff
@@ -0,0 +1,192 @@
+// Copyright 2020 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+use aes_crate::{Aes128, Aes192, Aes256, BlockCipher, NewBlockCipher};
```

Comment:
I tried upgrading, but this seems non-trivial. The `block_modes` crate was deprecated, as you pointed out, some traits were removed and instead `crypto-common` should now be used. Since the upgrade requires more than just replacing some imports, it seems beyond my expertise to do the upgrade. Personally, I'd be fine with getting the current version shipped and upgrading at a later point.